pkg/server/service.go

```diff
@@ -88,15 +91,17 @@ type criContainerdService struct {
 	// imageStoreService is the containerd service to store and track
 	// image metadata.
 	imageStoreService images.Store
+	// netPlugin is used to setup and teardown network when run/stop pod sandbox.
+	netPlugin ocicni.CNIPlugin
 }
 
 // NewCRIContainerdService returns a new instance of CRIContainerdService
-func NewCRIContainerdService(conn *grpc.ClientConn, rootDir string) CRIContainerdService {
+func NewCRIContainerdService(conn *grpc.ClientConn, opt *options.CRIContainerdOptions) (CRIContainerdService, error) {
```

Comment:
I prefer passes in parameters one by one, or else in the future it will be hard to figure out which option is used in containerd service, which is not.